schemas/job_schema.json

```diff
@@ -146,6 +146,8 @@
             "hiveJDBCURL": { "type": "string" },
             "hiveUserName": { "type": "string" },
             "hivePassword": { "type": "string" },
+            "manualHiveSync": { "type": "boolean"},
```

Comment:
Also add it to the config.yaml

Reply:
it is passed through the config.yaml, but needs to be a part of Hudi object
I don't understand what you mean

Reply:
There's a file in a folder called config/job.yaml in metorikku project, update there as well (documentation), also add some comments and explain what this does